VariableDeclarationUsageDistance: no violation to move variables closer to block

```
$ cat TestClass.java
public class TestClass {
    void method() {
        int a = 3;
        int b = 2;

        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();

        for (int i = 0; i < 10; i++) {
            a = a + b;
        }
    }

    public void run() {};
}

$ cat TestConfig.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
<module name="VariableDeclarationUsageDistance">
<property name="allowedDistance" value="1" />
<property name="validateBetweenScopes" value="false" />
</module>
    </module>
</module>

$ java -jar checkstyle-7.1.2-all.jar -c TestConfig.xml TestClass.java
Starting audit...
Audit done.
```

I use `VariableDeclarationUsageDistance` to make sure my variables are close to where they are being used. It makes reading the code easier most times.
In the example, there are no violations that variables a and b should be closer to the block that uses them. They are not very close to the block at all, and are not declared `final`. I am expecting them to put out violations since they are so far away, just like if the for loop didn't exist.
On the [XDoc](http://checkstyle.sourceforge.net/config_coding.html#VariableDeclarationUsageDistance) in the `Notes` section, I understand from case 1 that the check will not require me to put variables inside the block. I am just asking that the variables be placed close enough to the block to satisfy the distance, assuming the block in question is considered 1 unit as a whole.

From the example above, I am expecting the following violations:
```
[ERROR] TestClass.java:3: Distance between variable 'a' declaration and its first usage is 13, but allowed 1.  Consider to make that variable as final if you still need to store its value in advance (before method calls that might do side effect on original value). [VariableDeclarationUsageDistance]
[ERROR] TestClass.java:4: Distance between variable 'b' declaration and its first usage is 13, but allowed 1.  Consider to make that variable as final if you still need to store its value in advance (before method calls that might do side effect on original value). [VariableDeclarationUsageDistance]
```

I am only expecting no violations with the config, if the code is defined as the following:

```
    void method() {
        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();

        int a = 3;
        int b = 2;

        for (int i = 0; i < 10; i++) {
            a = a + b;
        }
    }
```

I cannot turn on validateBetweenScopes because it acts as if the braces do not exist and with a distance of 1 it will ask for variables to be moved inside the braces which interfers with loops.

@romani I am closing this issue.
Somehow I missed the property `validateBetweenScopes` which solves this issue. I guess I accidentally assumed it would be on by default.

````
$ cat TestClass.java
public class TestClass {
    void method() {
        int a = 3;
        int b = 2;

        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();

        for (int i = 0; i < 10; i++) {
            a = a + b;
        }
    }

    public void run() {};
}

$ cat TestConfig.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
<module name="VariableDeclarationUsageDistance">
<property name="allowedDistance" value="1" />
<property name="validateBetweenScopes" value="true" />
</module>
    </module>
</module>

$ java -jar checkstyle-7.6-all.jar -c TestConfig.xml TestClass.java
Starting audit...
[ERROR] TestClass.java:3: Distance between variable 'a' declaration and its first usage is 13, but allowed 1.  Consider to make that variable as final if you still need to store its value in advance (before method calls that might do side effect on original value). [VariableDeclarationUsageDistance]
[ERROR] TestClass.java:4: Distance between variable 'b' declaration and its first usage is 13, but allowed 1.  Consider to make that variable as final if you still need to store its value in advance (before method calls that might do side effect on original value). [VariableDeclarationUsageDistance]
Audit done.
Checkstyle ends with 2 errors.
````

@romani I am expecting a violation with "validateBetweenScopes=false".
I cannot turn on `validateBetweenScopes` because it acts as if the braces do not exist and with a distance of `1` it will ask for variables to be moved inside the braces.
Plus there is already a violation with "validateBetweenScopes=true". Here is an example:
````
$ cat TestClass.java
public class TestClass {
    void method() {
        int a = 3;

        for (int i = 0; i < 10; i++) {
            run();

            a = a + 1;
        }
    }
}

$ cat TestConfig.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
<module name="VariableDeclarationUsageDistance">
<property name="allowedDistance" value="1" />
<property name="validateBetweenScopes" value="true" />
</module>
    </module>
</module>

$ java -jar checkstyle-7.6-all.jar -c TestConfig.xml TestClass.java
Starting audit...
[ERROR] TestClass.java:3: Distance between variable 'a' declaration and its first usage is 2, but allowed 1.  Consider to make that variable as final if you still need to store its value in advance (before method calls that might do side effect on original value). [VariableDeclarationUsageDistance]
Audit done.
Checkstyle ends with 1 errors.
````

> I cannot turn on validateBetweenScopes because it acts as if the braces do not exist and with a distance of 1 it will ask for variables to be moved inside the braces which interfers with loops.


my problem is here when I turn on the property it reported the same violation that you expected and nothing asked for variables to be moved inside the braces

```
PS D:\test> cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
<module name="VariableDeclarationUsageDistance">
<property name="allowedDistance" value="1" />
<property name="validateBetweenScopes" value="true" />  <-- I turned it on
</module>
    </module>
</module>
PS D:\test> cat src/Test.java
public class Test {
     void method() {
        int a = 3;
        int b = 2;

        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();
        run();

        for (int i = 0; i < 10; i++) {
            a = a + b;
        }
    }

    public void run() {};
}
PS D:\test> java -jar checkstyle-10.13.0-all.jar -c config.xml  src/Test.java
Starting audit...
[ERROR] D:\test\src\Test.java:3:9: Distance between variable 'a' declaration and its first usage is 13, but allowed 1.  Consider making that variable final if you still need to store its value in advance (before method calls that might have side effects on the original value). [VariableDeclarationUsageDistance]
[ERROR] D:\test\src\Test.java:4:9: Distance between variable 'b' declaration and its first usage is 13, but allowed 1.  Consider making that variable final if you still need to store its value in advance (before method calls that might have side effects on the original value). [VariableDeclarationUsageDistance]
Audit done.
Checkstyle ends with 2 errors.

```
So I am confused on this point if we turn on validateBetweenScopes we will get your required output and I think that it also should not raise a violation if it is off we should not calculate the distance between the declaration of the variable and its first usage in the different scopes. according to the [doc](https://checkstyle.org/checks/coding/variabledeclarationusagedistance.html#VariableDeclarationUsageDistance)

@rnveach please correct me if I am missing something

--- a/config/pitest-suppressions/pitest-coding-1-suppressions.xml
+++ b/config/pitest-suppressions/pitest-coding-1-suppressions.xml
@@ -1,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressedMutations>
+  <mutation unstable="false">
+    <sourceFile>VariableDeclarationUsageDistanceCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.coding.VariableDeclarationUsageDistanceCheck</mutatedClass>
+    <mutatedMethod>getDistToVariableUsageInChildNode</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.NakedReceiverMutator</mutator>
+    <description>replaced call to com/puppycrawl/tools/checkstyle/api/DetailAST::getFirstChild with receiver</description>
+    <lineContent>examineNode = examineNode.getFirstChild().getNextSibling();</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>VariableDeclarationUsageDistanceCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.coding.VariableDeclarationUsageDistanceCheck</mutatedClass>
+    <mutatedMethod>getDistToVariableUsageInChildNode</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.NakedReceiverMutator</mutator>
+    <description>replaced call to com/puppycrawl/tools/checkstyle/api/DetailAST::getNextSibling with receiver</description>
+    <lineContent>examineNode = examineNode.getFirstChild().getNextSibling();</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>VariableDeclarationUsageDistanceCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.coding.VariableDeclarationUsageDistanceCheck</mutatedClass>
+    <mutatedMethod>getDistToVariableUsageInChildNode</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.NonVoidMethodCallMutator</mutator>
+    <description>removed call to com/puppycrawl/tools/checkstyle/api/DetailAST::getType</description>
+    <lineContent>if (examineNode.getType() == TokenTypes.LABELED_STAT) {</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>VariableDeclarationUsageDistanceCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.coding.VariableDeclarationUsageDistanceCheck</mutatedClass>
+    <mutatedMethod>getDistToVariableUsageInChildNode</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.RemoveConditionalMutator_EQUAL_ELSE</mutator>
+    <description>removed conditional - replaced equality check with false</description>
+    <lineContent>if (examineNode.getType() == TokenTypes.LABELED_STAT) {</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>VariableDeclarationUsageDistanceCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.coding.VariableDeclarationUsageDistanceCheck</mutatedClass>
+    <mutatedMethod>getDistToVariableUsageInChildNode</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.RemoveSwitchMutator_4</mutator>
+    <description>RemoveSwitch 4 (case value 89)</description>
+    <lineContent>switch (examineNode.getType()) {</lineContent>
+  </mutation>
+</suppressedMutations>

--- a/src/it/java/com/google/checkstyle/test/chapter4formatting/rule4822variabledistance/VariableDeclarationUsageDistanceTest.java
+++ b/src/it/java/com/google/checkstyle/test/chapter4formatting/rule4822variabledistance/VariableDeclarationUsageDistanceTest.java
@@ -43,6 +43,7 @@ public class VariableDeclarationUsageDistanceTest extends AbstractGoogleModuleTe
             "219:9: " + getCheckMessage(clazz, msgExt, "t", 5, 3),
             "483:9: " + getCheckMessage(clazz, msgExt, "myOption", 7, 3),
             "495:9: " + getCheckMessage(clazz, msgExt, "myOption", 6, 3),
+            "508:9: " + getCheckMessage(clazz, msgExt, "count", 4, 3),
         };
 
         final Configuration checkConfig =

--- a/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4822variabledistance/InputVariableDeclarationUsageDistanceCheck.java
+++ b/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4822variabledistance/InputVariableDeclarationUsageDistanceCheck.java
@@ -505,7 +505,7 @@ public class InputVariableDeclarationUsageDistanceCheck {
     public int testIssue32_11(String toDir)
             throws Exception
     {
-        int count = 0;
+        int count = 0;  // warn
         String[] files = {};
 
         System.identityHashCode("Data archival started");

--- a/src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheck.java
+++ b/src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheck.java
@@ -306,7 +306,7 @@ public class VariableDeclarationUsageDistanceCheck extends AbstractCheck {
         while (!firstUsageFound && currentAst != null) {
             if (currentAst.getFirstChild() != null) {
                 if (isChild(currentAst, variableIdentAst)) {
-                    dist = getDistToVariableUsageInChildNode(currentAst, variableIdentAst, dist);
+                    dist = getDistToVariableUsageInChildNode(currentAst, dist);
                     variableUsageAst = currentAst;
                     firstUsageFound = true;
                 }
@@ -324,11 +324,10 @@ public class VariableDeclarationUsageDistanceCheck extends AbstractCheck {
      * Returns the distance to variable usage for in the child node.
      *
      * @param childNode child node.
-     * @param varIdent variable variable identifier.
      * @param currentDistToVarUsage current distance to the variable usage.
      * @return the distance to variable usage for in the child node.
      */
-    private static int getDistToVariableUsageInChildNode(DetailAST childNode, DetailAST varIdent,
+    private static int getDistToVariableUsageInChildNode(DetailAST childNode,
                                                          int currentDistToVarUsage) {
         DetailAST examineNode = childNode;
         if (examineNode.getType() == TokenTypes.LABELED_STAT) {
@@ -336,6 +335,7 @@ public class VariableDeclarationUsageDistanceCheck extends AbstractCheck {
         }
 
         int resultDist = currentDistToVarUsage;
+
         switch (examineNode.getType()) {
             case TokenTypes.SLIST:
                 resultDist = 0;
@@ -345,17 +345,12 @@ public class VariableDeclarationUsageDistanceCheck extends AbstractCheck {
             case TokenTypes.LITERAL_DO:
             case TokenTypes.LITERAL_IF:
             case TokenTypes.LITERAL_SWITCH:
-                if (isVariableInOperatorExpr(examineNode, varIdent)) {
-                    resultDist++;
-                }
-                else {
-                    // variable usage is in inner scope
-                    // reset counters, because we can't determine distance
-                    resultDist = 0;
-                }
+                // variable usage is in inner scope, treated as 1 block
+                // or in operator expression, then distance + 1
+                resultDist++;
                 break;
             default:
-                if (examineNode.findFirstToken(TokenTypes.SLIST) == null) {
+                if (childNode.findFirstToken(TokenTypes.SLIST) == null) {
                     resultDist++;
                 }
                 else {
@@ -706,22 +701,6 @@ public class VariableDeclarationUsageDistanceCheck extends AbstractCheck {
                 break;
             }
             ast = ast.getNextSibling();
-        }
-
-        // Variable may be met in ELSE declaration
-        // So, check variable usage in these declarations.
-        if (!isVarInOperatorDeclaration) {
-            final DetailAST elseBlock = operator.getLastChild();
-
-            if (elseBlock.getType() == TokenTypes.LITERAL_ELSE) {
-                // Get IF followed by ELSE
-                final DetailAST firstNodeInsideElseBlock = elseBlock.getFirstChild();
-
-                if (firstNodeInsideElseBlock.getType() == TokenTypes.LITERAL_IF) {
-                    isVarInOperatorDeclaration =
-                        isVariableInOperatorExpr(firstNodeInsideElseBlock, variable);
-                }
-            }
         }
 
         return isVarInOperatorDeclaration;

--- a/src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java
+++ b/src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java
@@ -301,7 +301,6 @@ public final class InlineConfigParser {
 
     private static String getFullyQualifiedClassName(String filePath, String moduleName)
             throws CheckstyleException {
-        String fullyQualifiedClassName;
         // This is a hack until https://github.com/checkstyle/checkstyle/issues/13845
         final Map<String, String> moduleMappings = new HashMap<>();
         moduleMappings.put("ParameterNumber",
@@ -320,6 +319,7 @@ public final class InlineConfigParser {
                 "com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck");
         moduleMappings.put("LineLength",
                 "com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck");
+        String fullyQualifiedClassName;
         if (moduleMappings.containsKey(moduleName)) {
             fullyQualifiedClassName = moduleMappings.get(moduleName);
         }

--- a/src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheckTest.java
+++ b/src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheckTest.java
@@ -168,6 +168,8 @@ public class VariableDeclarationUsageDistanceCheckTest extends
             "231:9: " + getCheckMessage(MSG_KEY, "t", 5, 1),
             "234:9: " + getCheckMessage(MSG_KEY, "c", 3, 1),
             "235:9: " + getCheckMessage(MSG_KEY, "d2", 3, 1),
+            "272:9: " + getCheckMessage(MSG_KEY, "sel", 2, 1),
+            "273:9: " + getCheckMessage(MSG_KEY, "model", 2, 1),
             "312:9: " + getCheckMessage(MSG_KEY, "wh", 2, 1),
             "355:9: " + getCheckMessage(MSG_KEY, "green", 2, 1),
             "356:9: " + getCheckMessage(MSG_KEY, "blue", 3, 1),
@@ -178,10 +180,20 @@ public class VariableDeclarationUsageDistanceCheckTest extends
             "483:9: " + getCheckMessage(MSG_KEY, "l2", 2, 1),
             "491:9: " + getCheckMessage(MSG_KEY, "myOption", 7, 1),
             "503:9: " + getCheckMessage(MSG_KEY, "myOption", 6, 1),
+            "516:9: " + getCheckMessage(MSG_KEY, "count", 4, 1),
             "517:9: " + getCheckMessage(MSG_KEY, "files", 2, 1),
             "552:13: " + getCheckMessage(MSG_KEY, "id", 2, 1),
             "554:13: " + getCheckMessage(MSG_KEY, "parentId", 4, 1),
+            "855:9: " + getCheckMessage(MSG_KEY, "a", 5, 1),
+            "867:9: " + getCheckMessage(MSG_KEY, "a", 5, 1),
+            "879:9: " + getCheckMessage(MSG_KEY, "a", 5, 1),
+            "891:9: " + getCheckMessage(MSG_KEY, "a", 5, 1),
+            "903:9: " + getCheckMessage(MSG_KEY, "a", 5, 1),
+            "913:9: " + getCheckMessage(MSG_KEY, "a", 5, 1),
+            "924:9: " + getCheckMessage(MSG_KEY, "a", 2, 1),
+            "945:9: " + getCheckMessage(MSG_KEY, "a", 5, 1),
             "990:9: " + getCheckMessage(MSG_KEY, "a", 2, 1),
+            "1001:9: " + getCheckMessage(MSG_KEY, "a", 4, 1),
             "1036:9: " + getCheckMessage(MSG_KEY, "c", 4, 1),
             "1066:9: " + getCheckMessage(MSG_KEY, "a", 4, 1),
         };
@@ -253,11 +265,19 @@ public class VariableDeclarationUsageDistanceCheckTest extends
             "231:9: " + getCheckMessage(MSG_KEY_EXT, "t", 5, 3),
             "491:9: " + getCheckMessage(MSG_KEY_EXT, "myOption", 7, 3),
             "503:9: " + getCheckMessage(MSG_KEY_EXT, "myOption", 6, 3),
+            "516:9: " + getCheckMessage(MSG_KEY_EXT, "count", 4, 3),
             "554:13: " + getCheckMessage(MSG_KEY_EXT, "parentId", 4, 3),
+            "855:9: " + getCheckMessage(MSG_KEY_EXT, "a", 5, 3),
+            "867:9: " + getCheckMessage(MSG_KEY_EXT, "a", 5, 3),
+            "879:9: " + getCheckMessage(MSG_KEY_EXT, "a", 5, 3),
+            "891:9: " + getCheckMessage(MSG_KEY_EXT, "a", 5, 3),
+            "903:9: " + getCheckMessage(MSG_KEY_EXT, "a", 5, 3),
+            "913:9: " + getCheckMessage(MSG_KEY_EXT, "a", 5, 3),
+            "945:9: " + getCheckMessage(MSG_KEY_EXT, "a", 5, 3),
             "1036:9: " + getCheckMessage(MSG_KEY_EXT, "c", 4, 3),
+            "1001:9: " + getCheckMessage(MSG_KEY_EXT, "a", 4, 3),
             "1066:9: " + getCheckMessage(MSG_KEY_EXT, "a", 4, 3),
         };
-
         verifyWithInlineConfigParser(
                 getPath("InputVariableDeclarationUsageDistanceDefault.java"), expected);
     }
@@ -283,8 +303,9 @@ public class VariableDeclarationUsageDistanceCheckTest extends
 
     @Test
     public void testLabels() throws Exception {
-        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
-
+        final String[] expected = {
+            "15:9: " + getCheckMessage(MSG_KEY_EXT, "eol", 5, 3),
+        };
         verifyWithInlineConfigParser(
                 getPath("InputVariableDeclarationUsageDistanceLabels.java"), expected);
     }
@@ -376,5 +397,15 @@ public class VariableDeclarationUsageDistanceCheckTest extends
 
         verifyWithInlineConfigParser(
                 getPath("InputVariableDeclarationUsageDistance1.java"), expected);
+    }
+
+    @Test
+    public void testVariableDeclarationUsageDistanceCloseToBlock() throws Exception {
+        final String[] expected = {
+            "15:9: " + getCheckMessage(MSG_KEY, "a", 13, 1),
+            "16:9: " + getCheckMessage(MSG_KEY, "b", 13, 1),
+        };
+        verifyWithInlineConfigParser(
+                getPath("InputVariableDeclarationUsageDistanceCloseToBlock.java"), expected);
     }
 }

--- a/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceCloseToBlock.java
+++ b/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceCloseToBlock.java
@@ -1,0 +1,59 @@
+/*
+VariableDeclarationUsageDistance
+allowedDistance = 1
+ignoreVariablePattern = (default)
+validateBetweenScopes = false
+ignoreFinal = false
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.variabledeclarationusagedistance;
+
+public class InputVariableDeclarationUsageDistanceCloseToBlock {
+    void method() {
+        int a = 3; // violation 'Distance between .* declaration and its first usage is 13.'
+        int b = 2; // violation 'Distance between .* declaration and its first usage is 13.'
+
+        run();
+        run();
+        run();
+        run();
+        run();
+        run();
+        run();
+        run();
+        run();
+        run();
+        run();
+        run();
+
+        for (int i = 0; i < 10; i++) {
+            a = a + b;
+        }
+    }
+
+     void method2() {
+        run();
+        run();
+        run();
+        run();
+        run();
+        run();
+        run();
+        run();
+        run();
+        run();
+        run();
+        run();
+
+        int a = 3;
+        int b = 2;
+
+        for (int i = 0; i < 10; i++) {
+            a = a + b;  // distance = 1
+        }
+    }
+
+    public void run() {};
+}

--- a/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceDefault.java
+++ b/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceDefault.java
@@ -513,7 +513,7 @@ public class InputVariableDeclarationUsageDistanceDefault {
     public int testIssue32_11(String toDir)
             throws Exception
     {
-        int count = 0;
+        int count = 0;  // violation 'Distance .* is 4.'
         String[] files = {};
 
         System.identityHashCode("Data archival started");
@@ -852,7 +852,7 @@ public class InputVariableDeclarationUsageDistanceDefault {
 
 class New6 {
     void a() {
-        int a = 1;
+        int a = 1; // violation 'Distance .* is 5.'
         System.lineSeparator();
         System.lineSeparator();
         System.lineSeparator();
@@ -864,7 +864,7 @@ class New6 {
     }
 
     void b() {
-        int a = 1;
+        int a = 1; // violation 'Distance .* is 5.'
         System.lineSeparator();
         System.lineSeparator();
         System.lineSeparator();
@@ -876,7 +876,7 @@ class New6 {
     }
 
     void c() {
-        int a = 1;
+        int a = 1; // violation 'Distance .* is 5.'
         System.lineSeparator();
         System.lineSeparator();
         System.lineSeparator();
@@ -888,7 +888,7 @@ class New6 {
     }
 
     void d() {
-        int a = 1;
+        int a = 1; // violation 'Distance .* is 5.'
         System.lineSeparator();
         System.lineSeparator();
         System.lineSeparator();
@@ -900,7 +900,7 @@ class New6 {
     }
 
     void f() {
-        int a = 1;
+        int a = 1; // violation 'Distance .* is 5.'
         System.lineSeparator();
         System.lineSeparator();
         System.lineSeparator();
@@ -910,7 +910,7 @@ class New6 {
     }
 
     void h() {
-        int a = 1;
+        int a = 1; // violation 'Distance .* is 5.'
         System.lineSeparator();
         System.lineSeparator();
         System.lineSeparator();
@@ -942,7 +942,7 @@ class New6 {
     }
 
     void k() {
-        int a = 1;
+        int a = 1; // violation 'Distance .* is 5.'
         System.lineSeparator();
         System.lineSeparator();
         System.lineSeparator();
@@ -998,7 +998,7 @@ class New6 {
     }
 
     void test() {
-        int a = 0;
+        int a = 0; // violation 'Distance .* is 4.'
 
         System.lineSeparator();
         System.lineSeparator();

--- a/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceLabels.java
+++ b/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceLabels.java
@@ -12,7 +12,7 @@ package com.puppycrawl.tools.checkstyle.checks.coding.variabledeclarationusagedi
 
 public class InputVariableDeclarationUsageDistanceLabels {
     public void method() {
-        boolean eol = false;
+        boolean eol = false; // violation 'Distance .* is 5.'
 
         nothing();
         nothing();

--- a/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceScopes.java
+++ b/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceScopes.java
@@ -269,8 +269,8 @@ public class InputVariableDeclarationUsageDistanceScopes {
     }
 
     public int[] getSelectedIndices() {
-        int[] sel = new int[5];
-        String model = "";
+        int[] sel = new int[5]; // violation 'Distance .* is 2.'
+        String model = ""; // violation 'Distance between .* declaration and its first usage is 2.'
         int a = 0;
         a++;
         for (int index = 0; index < 5; ++index) {
@@ -513,7 +513,7 @@ public class InputVariableDeclarationUsageDistanceScopes {
     public int testIssue32_11(String toDir)
             throws Exception
     {
-        int count = 0;
+        int count = 0; // violation 'Distance .* is 4.'
         String[] files = {}; // violation 'Distance .* is 2.'
 
         System.identityHashCode("Data archival started");
@@ -852,7 +852,7 @@ public class InputVariableDeclarationUsageDistanceScopes {
 
 class New4 {
     void a() {
-        int a = 1;
+        int a = 1; // violation 'Distance .* is 5.'
         System.lineSeparator();
         System.lineSeparator();
         System.lineSeparator();
@@ -864,7 +864,7 @@ class New4 {
     }
 
     void b() {
-        int a = 1;
+        int a = 1; // violation 'Distance .* is 5.'
         System.lineSeparator();
         System.lineSeparator();
         System.lineSeparator();
@@ -876,7 +876,7 @@ class New4 {
     }
 
     void c() {
-        int a = 1;
+        int a = 1; // violation 'Distance .* is 5.'
         System.lineSeparator();
         System.lineSeparator();
         System.lineSeparator();
@@ -888,7 +888,7 @@ class New4 {
     }
 
     void d() {
-        int a = 1;
+        int a = 1; // violation 'Distance .* is 5.'
         System.lineSeparator();
         System.lineSeparator();
         System.lineSeparator();
@@ -900,7 +900,7 @@ class New4 {
     }
 
     void f() {
-        int a = 1;
+        int a = 1; // violation 'Distance .* is 5.'
         System.lineSeparator();
         System.lineSeparator();
         System.lineSeparator();
@@ -910,7 +910,7 @@ class New4 {
     }
 
     void h() {
-        int a = 1;
+        int a = 1; // violation 'Distance .* is 5.'
         System.lineSeparator();
         System.lineSeparator();
         System.lineSeparator();
@@ -921,7 +921,7 @@ class New4 {
     }
 
     void i() {
-        int a = 1;
+        int a = 1; // violation 'Distance .* is 2.'
         switch (Math.max(1, 2)) {
         case 1:
             System.lineSeparator();
@@ -942,7 +942,7 @@ class New4 {
     }
 
     void k() {
-        int a = 1;
+        int a = 1; // violation 'Distance .* is 5.'
         System.lineSeparator();
         System.lineSeparator();
         System.lineSeparator();
@@ -998,7 +998,7 @@ class New4 {
     }
 
     void test() {
-        int a = 0;
+        int a = 0; // violation 'Distance between .* declaration and its first usage is 4.'
 
         System.lineSeparator();
         System.lineSeparator();

